src/aind_data_asset_indexer/utils.py

```diff
@@ -11,7 +11,11 @@
 
 from aind_codeocean_api.codeocean import CodeOceanClient
 from aind_data_schema.core.data_description import DataLevel, DataRegex
-from aind_data_schema.core.metadata import ExternalPlatforms, Metadata
+from aind_data_schema.core.metadata import (
```

Comment:
There's a TODO in Line 30 for `core_schema_file_names`. Should I refactor to use the static `CORE_FILES` list in `aind_data_schema.core.metadata`?

Reply:
Sounds good. It might require a little bit of refactoring.

Reply:
This is done. Also updated `core_schema_file_names` as a map of `field_name` to `file_name` to avoid converting between those repeatedly.